notebooks/source/ordinalregression.ipynb

```diff
@@ -0,0 +1,239 @@
+    "\n",
+    "This poses a challenge for statistical modeling as the data do not fit the most well known modelling approaches (e.g. linear regression). Modeling the data as [categorical](https://en.wikipedia.org/wiki/Categorical_distribution) is one possibility, but it disregards the inherent ordering in the data, and may be less statistically efficient. There are multiple appoaches for modeling ordered data. Here we will show how to use the OrderedLogistic distribution using cutpoints that are sampled from a Normal distribution with as additional constrain that the cutpoints they are ordered. For a more in-depth discussion of Bayesian modeling of ordinal data, see e.g. [Michael Betancour's blog](https://betanalpha.github.io/assets/case_studies/ordinal_regression.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jax import numpy as np, random\n",
+    "import numpyro\n",
+    "from numpyro import sample\n",
+    "from numpyro.distributions import constraints, Normal, ImproperUniform, Categorical, OrderedLogistic\n",
+    "from numpyro.infer.mcmc import MCMC\n",
+    "from numpyro.infer.hmc import NUTS\n",
+    "import pandas as pd\n",
+    "import seaborn as sns"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, generate data some with ordinal structure"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "value counts of Y:\n",
+      "1    19\n",
+      "2    16\n",
+      "0    15\n",
+      "Name: Y, dtype: int64\n",
+      "mean(X) for Y == 0: 0.042\n",
+      "mean(X) for Y == 1: 0.832\n",
+      "mean(X) for Y == 2: 1.448\n"
+     ]
+    }
+   ],
+   "source": [
+    "simkeys  = random.split(random.PRNGKey(1), 2)\n",
+    "nsim     = 50\n",
+    "nclasses = 3\n",
+    "Y        = Categorical(logits=np.zeros(nclasses)).sample(simkeys[0], sample_shape=(nsim,))\n",
+    "X        = Normal().sample(simkeys[1], sample_shape = (nsim,))\n",
+    "X       += Y\n",
+    "\n",
+    "print(\"value counts of Y:\")\n",
+    "df = pd.DataFrame({'X': X, 'Y': Y})\n",
+    "print(df.Y.value_counts())\n",
+    "\n",
+    "for i in range(nclasses):\n",
+    "    print(f\"mean(X) for Y == {i}: {X[np.where(Y==i)].mean():.3f}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<AxesSubplot:xlabel='Y', ylabel='X'>"
```

Comment:
I think you can add `;` at the end of the code `sns.violinplot(x='Y', y='X', data=df)` to remove this text.